source/app/store/entities/tracks/data.js

```diff
@@ -35,14 +35,38 @@ of document digitization and extraction of sensitive information.
 
 For demo purposes, the solution is pre-loaded with data related to certain medical conditions such as Diabetes and Kidney Disease. Ask a question related to these topics or use the suggested queries in the search bar and explore the different search capabilities.
 `
+// Kendra without comprehend Medical
```

Comment:
Similar to a previous comment, I'd like to reduce the redundancy. Should we consider conditional updates for text? Lets discuss with the team to see what method fits best.

Reply:
Yeah I think that would be a better solution. We can list the respective services inside the text depending on what all flags are set to true.